### ISSUE TITLE
Could we add the discriminator to the throttle_data?

### ISSUE THREAD

**Issue by Taytay** [kickstarter/rack-attack#420]
The `throttle_data` stored in `request.env['rack.attack.throttle_data']` currently consists of:
```ruby
        data = {
          count: count,
          period: current_period,
          limit: current_limit,
          epoch_time: epoch_time
        }
```
https://github.com/kickstarter/rack-attack/blob/master/lib/rack/attack/throttle.rb#L33-L38

It would be helpful for me if the discriminator for the current request were added as well:
```ruby
        data = {
          count: count,
          period: current_period,
          limit: current_limit,
          epoch_time: epoch_time,
          discriminator: discriminator
        }

        (request.env['rack.attack.throttle_data'] ||= {})[name] = data
```

I figured that no one else needed this data, or I am thinking about this logic so incorrectly, that what I'm asking for doesn't work. Would it be feasible to add this?

**Comment by grzuy** [comment]
Hi @Taytay,

Makes sense to me.

Thank you!

**Comment by grzuy** [comment]
@Taytay Something like #422?

**Comment by Taytay** [comment]
Lol - yeah, something like that! Thank you for the quick and helpful response. I’ve monkey patched in the meantime, but I look forward to reverting to the “real deal” you’ve got here!

**Comment by grzuy** [comment]
Hehe, no worries! :-)

**Comment by grzuy** [comment]
Released `v6.1.0`.
